packages/cli/src/tools/loadMetroConfig.ts

```diff
@@ -110,32 +119,22 @@ export const getDefaultConfig = (ctx: Config): MetroConfig => {
       ),
     },
     watchFolders: getWatchFolders(),
+    reporter: opts.reporter,
   };
 };
 
-export interface ConfigOptionsT {
-  maxWorkers?: number;
-  port?: number;
-  projectRoot?: string;
-  resetCache?: boolean;
-  watchFolders?: string[];
-  sourceExts?: string[];
-  reporter?: any;
-  config?: string;
-}
-
 /**
  * Loads Metro Config and applies `options` on top of the resolved config.
  *
  * This allows the CLI to always overwrite the file settings.
  */
 export default function load(
   ctx: Config,
-  options?: ConfigOptionsT,
+  opts: ConfigOptions = {},
 ): Promise<MetroConfig> {
-  const defaultConfig = getDefaultConfig(ctx);
-  if (options && options.reporter) {
-    defaultConfig.reporter = options.reporter;
```

Comment:
If `options.reporter` is not there, we're going to pass `reporter: undefined` to Metro, and if I remember correctly, it resulted in no reporters at all, instead of using a default reporter, because Metro checked for presence of keys, instead of their values. But maybe it's not the case anymore. Please test thoroughly

Reply:
Ok, thanks, will double check that one tomorrow!